traitlets/config/application.py

```diff
@@ -27,7 +27,9 @@
 from ipython_genutils.importstring import import_item
 from ipython_genutils.text import indent, wrap_paragraphs, dedent
 from ipython_genutils import py3compat
-from ipython_genutils.py3compat import string_types, iteritems
+
+import six
+from six import iteritems
```

Comment:
Yup. We aren't operating on any giant dicts here. We can afford to let Python 2 be the tiniest bit slower on what we are dealing with. No need to call in six for this one.